### DISCUSSION TITLE
Thread device commissioning via BLE vs External Commissioning via a Thread Border Router?

### DISCUSSION THREAD

**Discussion by lpfsue** [openthread/openthread#8094] [Q&A] [answered]
Does someone know why Matter (Project CHIP) uses Thread device commissioning via BLE instead of External Commissioning via a Thread Border Router?

From what I understand it would have also be possible to not use BLE at all (and not require the chip vendors to make all these chip variants which need to support BLE / Thread on the same PHY).

Would be great if someone has some idea why this is the case.

**Comment by jwhui** [comment, marked as answer]
When it comes to device provisioning, BLE has a couple of benefits. Because mobile devices generally support BLE, the mobile device can communicate directly with the joining device using a point-to-point connection over BLE. This ensures the mobile and joining device can transfer information whenever they are in physical proximity (which is the typical scenario when commissioning smart home devices). Also, if there are any issues with finding a Thread Border Router, that can also be provided as feedback to the mobile device using the BLE connection.
